evmjit/libevmjit/Compiler.cpp

```diff
@@ -49,6 +49,13 @@ void Compiler::createBasicBlocks(code_iterator _codeBegin, code_iterator _codeEn
 		return _curr + offset;
 	};
 
+	// Skip all STOPs in the end
+	for (; _codeEnd != _codeBegin; --_codeEnd)
+	{
```

Comment:
even though I also prefer it with the braces, the style basically says 1 line statement. This contains an if only, so it can be omitted. The second line the `break` belongs to the if and not to the for